scripts/code-examples.js

```diff
@@ -0,0 +1,133 @@
+const fs = require('fs-extra');
+const path = require('path');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+const rimraf = require('rimraf');
+
+const REPO_ROOT_DIR = __dirname.split('scripts')[0];
+const NEXT_EXAMPLES_DIR = path.join(REPO_ROOT_DIR, 'examples', 'next');
+const TMP_DIR_NAME = 'tmp';
+const TMP_DIR = path.join('examples', TMP_DIR_NAME);
+
+const HOT_WRAPPERS = ['@handsontable/react', '@handsontable/angular', '@handsontable/vue'];
+
+const [/* node bin */, /* path to this script */, shellCommand, hotVersion] = process.argv;
+
+// Function search recursively in the provided `dirPath` for the package.json file
+// and gets the project directory path.
+const getExamplesFolders = (dirPath, exampleFolders) => {
+  const files = fs.readdirSync(dirPath);
+
+  exampleFolders = exampleFolders || [];
+
+  files.forEach((file) => {
+    if (file !== 'node_modules' && fs.statSync(path.join(dirPath, file)).isDirectory()) {
+      exampleFolders = getExamplesFolders(path.join(dirPath, file), exampleFolders);
+      return;
+    }
+    if (file === 'package.json') {
+      exampleFolders.push(dirPath);
+    }
+  });
+
+  return exampleFolders;
+};
+
+const getHotWrapperName = (packageJson) => {
+  const { dependencies } = packageJson;
+
+  return HOT_WRAPPERS.find(wrapper => Object.keys(dependencies).find(dependency => dependency === wrapper));
+};
+
+const updatePackageJsonWithVersion = (projectDir, version) => {
+  const packageJsonPath = path.join(projectDir, 'package.json');
+  const packageJson = fs.readJsonSync(packageJsonPath);
+  const wrapper = getHotWrapperName(packageJson);
+
+  packageJson.version = version;
+  packageJson.dependencies.handsontable = version;
+  if (wrapper) {
+    // TODO: uncomment it when wrappers will be using the same versioning as Handsontable
+    // pJsonFile.dependencies[wrapper] = hotVersion;
+  }
+  fs.writeJsonSync(packageJsonPath, packageJson, { spaces: 2 });
+};
+
+const runNpmCommandInExample = (exampleDir, command) => {
+  // eslint-disable-next-line
+  console.log(chalk.yellow(`"${command}" STARTED IN DIRECTORY "${exampleDir}"`));
```

Comment:
I know this is nitpicking, but maybe the color of this message should not be yellow? Yellow messages in the console kind of look like warnings to me 🤔

Reply:
> Also, the `npm run examples:build next` fails on the react directory (unless I'm missing something):
> 
> ```
> The react-scripts package provided by Create React App requires a dependency:
> 
>   "babel-eslint": "^10.1.0"
> 
> Don't try to install it manually: your package manager does it automatically.
> However, a different version of babel-eslint was detected higher up in the tree:
> 
>   /home/rof/src/github.com/handsontable/handsontable/node_modules/babel-eslint (version: 11.0.0-beta.2) 
> 
> Manually installing incompatible versions is known to cause hard-to-debug issues.
> ```
> 
> This might be caused by the same dependencies being installed in the monorepo, so some tweaking might be needed?

And that's a problem. I removed the `.env` file with the`SKIP_PREFLIGHT_CHECK=true` and the build started failing. I can bring it back, but maybe you have other solutions?

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/68065e7f7bddd6ad51522f40837c32aa448b4cc6